tests/cross/src/test/scala/tests/pc/MacroCompletionSuite.scala

```diff
@@ -120,9 +148,7 @@ class MacroCompletionSuite extends BaseCompletionSuite {
     compat = Map(
       "2.11" ->
         """|fold[X](fa: Int => X, fb: String => X): X
-           |""".stripMargin,
-      // NOTE(tgodzik): the presentation compiler returns empty results here in 2.12.9
-      "2.13.0" -> ""
```

Comment:
It seems that the issues were finally fixed - probably because we are using a much more limited classpath now.